package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@google-cloud/vision",
   "description": "Google Cloud Vision API client for Node.js",
-  "version": "0.20.0",
+  "version": "0.20.1",
```

Comment:
I'm fine with skipping formalities, but wanted to make sure this version bump was intentional (usually, they get their own PR with release notes). Just checking!

Reply:
this is an experiment :) addng the do not merge

Reply:
Yep it must not be merged :)  Sorry for doing it silently!

Reply:
To give some context, we're investigating behavior of `npm link` with the lock files which takes so much of our time.